setup.py

```diff
@@ -345,7 +348,7 @@ def run(self):
       package_dir = {'':'src'},
       packages=find_packages('src') + ['allmydata.test.plugins'],
       classifiers=trove_classifiers,
-      python_requires="<3.0",
+      python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, !=3.6.*, !=3.7.*, <4",
       install_requires=install_requires,
```

Comment:
maybe a comment here would be good (I think I parse that as "we support python 2.7 and 3.8 only").

That said, though, I think we SHOULD support python 3.6 because that is the highest python version that PyPy supports (AFAIK).

Reply:
> I think I parse that as "we support python 2.7 and 3.8 only"

3.8, 3.9, 3.10 ... but not 4.x or above. (I guess a theoretical 2.8 would pass, as well.) The example is straight from ... some doc that I can't find again right now. 🐭  &nbsp; I agree a comment is in order, once [I find that doc and] we settle on specifics ...

> highest version

Okay, soooo ...

```
python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <4"
```

Like that? Read as: "2.7 and then 3.6 or greater in the 3.x line."

Reply:
Decided on Thursday call to target 3.6 for initial port.